2024-10-02 23:43:12,040 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Configure stats pid to 43976
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Loading settings from /home/vishnu/.config/wandb/settings
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Loading settings from /home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/wandb/settings
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'mlp-wine-quality', 'entity': 'vishnuvarun-iiit-hyderabad', 'root_dir': '/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077', 'run_id': 'oltfghk8', 'sweep_param_path': '/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/wandb/sweep-hpyu9v69/config-oltfghk8.yaml', 'sweep_id': 'hpyu9v69'}
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'assignments/3/mlpc.py', 'program_abspath': '/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/assignments/3/mlpc.py', 'program': '/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/assignments/3/mlpc.py'}
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_init.py:_log_setup():532] Logging user logs to /home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/wandb/run-20241002_234312-oltfghk8/logs/debug.log
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_init.py:_log_setup():533] Logging internal logs to /home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/wandb/run-20241002_234312-oltfghk8/logs/debug-internal.log
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_init.py:init():617] calling init triggers
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_init.py:init():624] wandb.init called with sweep_config: {'activation': 'tanh', 'batch_size': 16, 'learning_rate': 0.09894754285897436, 'optimizer': 'sgd'}
config: {}
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_init.py:init():667] starting backend
2024-10-02 23:43:12,041 INFO    MainThread:43976 [wandb_init.py:init():671] sending inform_init request
2024-10-02 23:43:12,044 INFO    MainThread:43976 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-02 23:43:12,045 INFO    MainThread:43976 [wandb_init.py:init():684] backend started and connected
2024-10-02 23:43:12,051 INFO    MainThread:43976 [wandb_run.py:_config_callback():1394] config_cb None None {'activation': 'tanh', 'batch_size': 16, 'learning_rate': 0.09894754285897436, 'optimizer': 'sgd'}
2024-10-02 23:43:12,051 INFO    MainThread:43976 [wandb_init.py:init():779] updated telemetry
2024-10-02 23:43:12,055 INFO    MainThread:43976 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-02 23:43:12,388 INFO    MainThread:43976 [wandb_init.py:init():863] starting run threads in backend
2024-10-02 23:43:12,392 WARNING MainThread:43976 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 882, in init
    self._wl._global_run_stack.append(run)
  File "/home/vishnu/3rd year/SMAI/smai-m24-assignments-Vishnuvarun077/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 317, in __getattr__
    def __getattr__(self, name: str) -> Any:
KeyboardInterrupt
2024-10-02 23:43:12,393 WARNING MsgRouterThr:43976 [router.py:message_loop():77] message_loop has been closed
